### DISCUSSION TITLE
element-plus 组件上的 简写样式属性是根据什么来的，文档在哪，比如 mt-5 表示上外边距是5

### DISCUSSION THREAD

**Discussion by dong-lufei** [element-plus/element-plus#15148] [Q&A] [answered]
element-plus 组件上的 简写样式属性是根据什么来的，文档在哪，比如 mt-5 表示上外边距是5

**Comment by ryuhangyeong** [comment, marked as answer]
https://github.com/element-plus/element-plus/tree/dev/docs

UnoCSS (https://unocss.dev/)

Is this what you want?
